assets/src/scss/layout/_navbar.scss

```diff
@@ -292,9 +293,19 @@ a.nav-link:hover:before,
     padding: 0;
   }
 
+  &.has-children {
+    a {
+      padding-right: 36px;
```

Comment:
Can we use a variable? `$sp-4x` if I'm not mistaken... We also should use `padding-inline-end` instead to account for RTL sites

Reply:
We can also directly write `&.has-children a` instead of adding another level